crates/sncast/src/starknet_commands/account/mod.rs

```diff
@@ -34,18 +34,35 @@ pub enum Commands {
     Delete(Delete),
 }
 
+#[derive(ValueEnum, Clone, Debug)]
+pub enum AccountType {
+    Oz,
```

Comment:
Imo there is no point in using acronym, whole name would be more obvious and imo it does not add that much writing. Imo we should at least use one version, either `open_zeppelin` or `oz`

Reply:
I think there's a significant difference between typing `--type oz` and `--type open_zeppelin`. On the other hand nothing prevents us from having it more descriptive In the accounts file. If you strongly think we should stick with one format, I'd prefer the acronym.

Reply:
I don't see a problem with typing open_zeppelin. This is not a command you run very often, full name is more discoverable. Currently somebody has to know what `oz` means.

Reply:
What about leaving the acronym, but improving the description for `--help` to have something like this?
```
          Possible values:
          - oz:     OpenZeppelin account implementation
          - argent: Argent account implementation
```

Reply:
Sounds good